rust/src/database/migrate/mod.rs

```diff
@@ -0,0 +1,71 @@
+use std::path::{Path, PathBuf};
+use crate::cli::ServerConfig;
+use anyhow::Result;
+use log::info;
+use scylla::Session;
+use crate::database::new_session;
+
+pub async fn migrate(server_config: &ServerConfig, should_drop_keyspace: bool) -> Result<()>
+{
+    let session = new_session(server_config).await.unwrap();
+
+    if should_drop_keyspace {
+        drop_keyspace(&session, server_config.keyspace.as_str()).await;
+    }
+
+    create_keyspace(&session, &server_config.keyspace.as_str()).await;
+    create_tables(&session).await;
+
+    Ok(())
+}
+
+async fn create_tables(session: &Session) {
+    let current_path = std::env::current_dir().unwrap();
+    let file_path = Path::new("src/database/migrate/migrate.cql");
+    let full_path = current_path.join(file_path);
+
+    let raw_queries = read_file(full_path)
+        .await
+        .unwrap_or_else(|_| panic!("Could not read file"));
+
+    let queries = raw_queries.split(";")
+        .map(|query| query.trim())
+        .collect::<Vec<&str>>();
+
+
+    info!("Creating tables...");
+    for query in queries {
+        if query.is_empty() {
+            continue;
+        }
+
+        let prepared_query = session.prepare(query).await.unwrap();
+        session.execute(&prepared_query, []).await.unwrap();
```

Comment:
Preparing "CREATE TABLE" one-shot statements is not encouraged. This causes unnecessary load over the whole cluster, because preparation occurs at every connection (so not only every node, not only every shard, but literally every connection!). OTOH, using `query()` prepares on a single connection only, then executes there.